yarn.lock

```diff
@@ -16955,7 +16955,7 @@ remark-parse@^4.0.0:
     parse-entities "^1.0.2"
     repeat-string "^1.5.4"
     state-toggle "^1.0.0"
-    trim "0.0.1"
+    trim "^0.0.3"
```

Comment:
I think the changing only trim version without modifing remark-parse version cause a inconsistent.